src/openms/source/FORMAT/HANDLERS/MzIdentMLHandler.cpp

```diff
@@ -1002,8 +1007,8 @@ namespace OpenMS
                         mod = xl_db->getModification(mods[s], "", ResidueModification::C_TERM);
                       }
                     }
-                    acc = mod.getPSIMODAccession();
-                    name = mod.getId();
+                    acc = mod->getPSIMODAccession();
+                    name = mod->getId();
```

Comment:
hmm for some reason gcc doesnt like this:



/.../OpenMS/src/openms/source/FORMAT/HANDLERS/MzIdentMLHandler.cpp:1011:39: error: ‘mod’ may be used uninitialized in this function [-Werror=maybe-uninitialized]

```suggestion
                    if (mod != nullptr) name = mod->getId();
```

should help?